packages/net/http/payload.pony

```diff
@@ -243,3 +242,26 @@ class iso Payload
     end
 
     list
+
+  fun string(): String =>
+    var str = proto + status.string() + method + "\n"
```

Comment:
I think it's a good thing that you moved to-string conversion here from `httpget.pony`, however in doing so you also changed the implementation. For example, there are spaces between proto, status, and method in `httpget.pony`, which are no longer present in yours.

`httpget.pony` conversion adheres to the [HTTP/1.1 response][1] format. If you take a look to specification, there's also an example of a request which is different. Currently, both request and response are handled with the same `Payload` class, thus it should either have distinct methods `request_string()`, `response_string()` or detect it somehow (although I don't think it's possible in this case) and produce the string accordingly.

[1]: https://tools.ietf.org/html/rfc7230#page-8